src/js/components/Brick.js

```diff
@@ -0,0 +1,47 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
+
+export default class Brick extends Component {
+  render() {
+    var classes = [CLASS_ROOT];
+    classes.push('brick--' + this.props.width + '-' + this.props.height);
+
+    if (this.props.colorIndex) {
+      classes.push('background-color-index-' + this.props.colorIndex);
+    }
+    if (this.props.className) {
+      classes.push(this.props.className);
+    }
+
+    var label = <div className="brick--label"><span>{this.props.label}</span></div>;
+    if (this.props.route) {
```

Comment:
Changed `route` prop to `href` to avoid confusion. Also updated grommet-docs (https://github.com/grommet/grommet-docs/pull/12) to reflect the change.